packages/indexer-cli/src/commands/indexer/actions/get.ts

```diff
@@ -91,6 +103,22 @@ module.exports = {
           ? OrderDirection[orderDirection.toUpperCase() as keyof typeof OrderDirection]
           : OrderDirection.DESC
       }
+
+      switch (typeof first) {
+        case 'boolean':
+          // can only be `true` at this point
+          fetchFirst = 1
```

Comment:
Similar comment to above. I'd suggest defaulting to `100` to maintain a smooth transition from previous behavior.